exporters/jaeger-thrift/src/test/java/io/opentelemetry/exporter/jaeger/thrift/JaegerThriftIntegrationTest.java

```diff
@@ -46,6 +47,7 @@ class JaegerThriftIntegrationTest {
   @Container
   public static GenericContainer<?> jaegerContainer =
       new GenericContainer<>("ghcr.io/open-telemetry/opentelemetry-java/jaeger:1.32")
+          .withImagePullPolicy(PullPolicy.alwaysPull())
```

Comment:
do we overwrite the versioned docker image above?

Reply:
Oddly, I _think_ we do. The job definition lives [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docker-test-containers-daily.yml#L33-L34). We use the [docker cp](https://docs.docker.com/engine/reference/commandline/cp/) command which is documented with "the destination is overwritten with the source file’s contents". 

So I think we're treating the `jaeger:1.32` tag like an ephemeral `latest` tag.